xopt/tests/generators/sequential/test_neldermead.py

```diff
@@ -184,9 +184,9 @@ def test_simplex_agreement(self):
 
         # Results should be the same
         xbest = X.data.iloc[X.data["y"].argmin()]
-        assert np.isclose(xbest["x0"], result[0]) and np.isclose(xbest["x1"], result[1]), (
-            "Xopt Simplex does not match the vanilla one"
-        )
+        assert np.isclose(xbest["x0"], result[0]) and np.isclose(
```

Comment:
make sure your version of ruff is the same as the version used in the github actions to make these agree